CODEOWNERS

```diff
@@ -0,0 +1,73 @@
+# See also:
+# - https://github.com/blog/2392-introducing-code-owners
+# - https://help.github.com/articles/about-codeowners/
+
+# Each line is a file pattern followed by one or more owners.
+# Please feel free to add yourself to a module or create a new mapping.
+# Ideally each module should have two or more owners.
+
+# Code owners are automatically requested for review
+# when someone opens a pull request that modifies code that they own.
+# Later matches take precedence.
+
+etc/c/* @CyberShadow
+std/* @andralex
+std/algorithm/* @andralex @JackStouffer @wilzbach
+std/array.d @JackStouffer @wilzbach
+std/ascii.d @JackStouffer @wilzbach
+#std/base64.d
+std/bigint.d @Biotronic
+#std/bitmanip.d
+std/c/windows/* @CyberShadow
+#std/compiler.d
+#std/complex.d
+std/concurrency.d @MartinNowak
+#std/container/
```

Comment:
You can add me here as well. Though, I guess @schveiguy would want to be at the front of the list?